src/main/java/org/spongepowered/common/data/key/KeyRegistry.java

```diff
@@ -112,6 +112,7 @@ public static void registerKeys() {
         keyMap.put("item_enchantments", makeListKey(ItemEnchantment.class, of("ItemEnchantments")));
         keyMap.put("banner_patterns", makeListKey(PatternLayer.class, of("BannerPatterns")));
         keyMap.put("banner_base_color", makeListKey(DyeColor.class, of("BannerBaseColor")));
+        keyMap.put("elder", makeSingleKey(Boolean.TYPE, Value.class, of("Elder")));
```

Comment:
`!"elder".equalsIgnoreCase("ELDER_GUARDIAN")` which is why I was hoping that by testing with the proper methods (including manipulating the data through keys of the `ElderData` `DataManipulator`), you'd find that there's an issue with the key being registered here :wink: 

Basically, the keymap,  you're not putting the right field name for the key you're wanting to register: [`Keys.ELDER_GUARDIAN`](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/data/key/Keys.java#L214).